src/main/java/com/dre/brewery/configuration/annotation/OkaeriConfigFileOptions.java

```diff
@@ -0,0 +1,40 @@
+package com.dre.brewery.configuration.annotation;
+
+import com.dre.brewery.configuration.configurer.BreweryXConfigurer;
+import eu.okaeri.configs.configurer.Configurer;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface OkaeriConfigFileOptions {
```

Comment:
I would honestly rename this to something like `@BreweryConfig` and make it required on our every config.
This would also remove the need for the `getOkaeriConfigFileOptions` monstrosity...